.github/workflows/pr_scan.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         project: 'Adyen.xcodeproj'
         params: '-derivedDataPath ./DerivedData -enableCodeCoverage YES -skipPackagePluginValidation'
         scheme: 'AdyenUIHost'
-        destination: 'name=iPhone 14,OS=16.4'
+        destination: 'name=iPhone 14,OS=17.0.1'
```

Comment:
GitHub's Xcode 15.0.1 [supposed to](https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md#installed-sdks) come with iOS 17.0.
So we might need to have latest simulator installed with 
```
xcversion simulators --install="iOS ${version}"
```

Reply:
https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md

Reply:
The logs stated that `17.0.1` is available so I just chose the latest

Reply:
if it works it works 😆

My main point is - there are two approaches:
1) rely on GitHub - use `latest` Xcode and then not specify iOS version - and hope that this will cover all "modern" setups
2) take control in our hands - specify both Xcode and iOS version - will require manual updates of workflow evry now and then

Reply:
Yes I agree with using the latest version - Regarding the iOS version it's a decision between either updating the version manually regularly or updating snapshots regularly 😅